docs/source/models/gbd2017/causes/diarrheal_diseases/diarrheal_diseases.rst

```diff
@@ -48,15 +51,20 @@ severity for their age/sex/location/year.
    Add relevant detail about diarrheal diseases modeling process from
    the CoD Appendix.
 
+GBD Hierarchy
+-------------
+
+.. image:: DD_cause_hierarchy.svg
```

Comment:
In your diagram, both "severe diarrheal diseases" and "Guillain-Barre syndrome" have sequela id 667.  Probably one of these should be different?  Otherwise looks good.

Reply:
whoops. thank you!